docs/configuredgraphfactory.txt

```diff
@@ -0,0 +1,504 @@
+[[configuredgraphfactory]]
+ConfiguredGraphFactory
+----------------------
+
+The `ConfiguredGraphFactory` is an access point to your graphs, similar
+to the `JanusGraphFactory`. These graph factories provide static methods
+to open and close your graphs.
+
+[[overview]]
+Overview
+~~~~~~~~
+
+`JanusGraphFactory` is a class that provides an access point to your
+graphs by providing an entire Configuration each opening.
+
+`ConfiguredGraphFactory` provides an access to your graphs for which
+you have previously created configurations using the
+`ConfigurationManagementGraph`.
+
+`ConfigurationManagementGraph` allows you to manage graph configurations.
+
+`JanusGraphManager` is an internal server component that tracks
+graph references, provided your graphs are configured to use it.
+
+[[configuredgraphfactory-versus-janusgraphfactory]]
+ConfiguredGraphFactory versus JanusGraphFactory
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+However, there is an important distinction between these two graph
+factories:
+
+1.  The `ConfiguredGraphFactory` can only be used if you have configured
+your server to use the `ConfigurationGraphManagement` APIs at server
+start.
+
+The benefits of using the `ConfiguredGraphFactory` is that:
+
+1.  You only need to supply a `String` to access your graphs, as opposed
+to the `JanusGraphFactory`-- which requires you to specify information
+about the backend you wish to use when accessing a graph-- every time
+you open a graph.
+2.  Assuming your `ConfigurationGraphManagement` has not been configured
```

Comment:
I would reword this to be more generic.  

If your `ConfigurationGraphManagement` is configured with a distributed storage backend then your graph configurations are available to all `JanusGraph` nodes in your cluster.